deploy/packer/app/scripts/deploy.sh

```diff
@@ -39,21 +45,36 @@ sudo modprobe snd-aloop
 # Create dependency file
 sudo depmod
 
-relengapi_proxy_version=2.3.1
-
 # Pull images used for sidecar containers
-docker pull taskcluster/taskcluster-proxy:5.1.0
-docker pull taskcluster/livelog:v4
-docker pull taskcluster/dind-service:v4.0
+docker pull taskcluster/taskcluster-proxy:$taskcluster_proxy_version
+docker pull taskcluster/livelog:v$livelog_version
+docker pull taskcluster/dind-service:v$dind_service_version
 docker pull taskcluster/relengapi-proxy:$relengapi_proxy_version
 
+# install and configure taskcluster-worker-runner
+sudo curl --fail -L -o /usr/local/bin/start-worker https://github.com/taskcluster/taskcluster-worker-runner/releases/download/v$worker_runner_version/start-worker-linux-amd64
+sudo chmod +x /usr/local/bin/start-worker
+sudo bash -c 'cat > /etc/start-worker.yml <<EOF
+provider:
+    providerType: aws-provisioner
+worker:
+    implementation: docker-worker
+    path: /home/ubuntu/docker_worker
+    configPath: /home/ubuntu/worker.cfg
+EOF'
```

Comment:
Now that we're also supporting gcp, I'll need to make this conditional and do a different configuration for the different images (basically, legacy deployment + aws -> providerType: aws-provisioner, otherwise aws -> providerType: amazon and gcp -> providerType: google).  But, I can do that in a followup.